### PR TITLE
Automate Changeset Release through CI

### PR DESCRIPTION
**Pull request by tomoyuki-hiranuma**
I tried to automate the changeset release process with CI.

 In order to use the automation, you seem to need to install [this bot](https://github.com/apps/changeset-bot) to this repository. And it might be necessary to give the bot permissions to write and create PRs  in github settings.

And, I need your help because I have not been able to test with this repository and have been testing with my own repository, so please let me know if there are any problems. I hope we can solve them together.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -26,7 +26,10 @@ jobs:
           version: 8.5.1
 
       - name: install packages
-        run: pnpm install
+        run: pnpm install --frozen-lockfile
```
Comment (poteboy) [inline .github/workflows/main.yml:29]: I guess that `--frozen-lockfile` is set to true by default in a CI environment, so we don't need to specify it?
https://pnpm.io/cli/install

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -25,5 +25,11 @@ jobs:
...
 
-      - name: Build packages
-        run: pnpm build
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          publish: pnpm release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (poteboy) [inline .github/workflows/release.yml:34]: When I tried to register an environment variable with the name "GITHUB_TOKEN", I encountered an error message saying "Variable name cannot start with 'GITHUB_' prefix." Can you change this part to a different name?

File: .github/workflows/main.yml (hunk 1 above)
Comment (tomoyuki-hiranuma) [reply .github/workflows/main.yml]: I see 👍 
It's new learning for me.
I fixed it!!

[abca3c3](https://github.com/poteboy/kuma-ui/pull/99/commits/abca3c3756308f1f42957bc496f8eda58aef4b01)

File: .github/workflows/release.yml (hunk 2 above)
Comment (tomoyuki-hiranuma) [reply .github/workflows/release.yml:34]: Probably "GITHUB_TOKEN" does not need to be registered. It should be handled internally.
https://docs.github.com/ja/actions/security-guides/automatic-token-authentication

**Pull request merged**